packages/picasso/src/PageHamburger/PageHamburgerPortal.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useCallback, useContext } from 'react'
+import Portal from '@material-ui/core/Portal'
+
+import PageHamburgerContext, {
+  PageHamburgerContextProps,
+} from './PageHamburgerContext'
+import { getHamburgerContainer } from './utils'
+
+type Props = {
+  children: React.ReactNode
+}
+
+const PageHamburgerPortal = ({ children }: Props) => {
+  const { hamburgerId } =
+    useContext<PageHamburgerContextProps>(PageHamburgerContext)
```

Comment:
I saw that this context will be used multiple times so wdyt about

```suggestion
    usePageHamburgerContext()
```

like here: https://github.com//toptal/picasso/blob/2d76bbf2e1b565829145f3fd338ff86c9d198f6b/packages/picasso/src/PageSidebar/SidebarContextProvider.tsx#L45